package.json

```diff
@@ -0,0 +1,83 @@
+{
+    "name": "mendix-tooltip",
+    "widgetName": "Tooltip",
```

Comment:
name and WidgetName are the same, widgetName is in CamelCase while name is lowercase